source-academy/frontend
# Ruizhi2001

### Description


The changes in this PR include:

1. Added a new button at the top right to change between 'Source', 'Scheme', and 'Python'
2. Chapter Select changes depending on the type of language chosen

### Type of change


### How to test


1. Run your local Source Academy 
2. Change the language chosen at the top right
3. See the menu of the ChapterSelectButton change upon clicking it

### Checklist
